src/entities/Customer.java

```diff
@@ -203,32 +211,34 @@ private void listMovie(String cinemaname, ArrayList<Cinema> cinemaDB) {
 	}
     }
 
-    private void checkSeatAvailability(ArrayList<CInema> cinemaDB) {
-	System.out.println("Enter the name of the Cinema you would like to check seat availability for");
-	String cinemaname = reader.readLine();
-	System.out.println("Enter the Movie for which you would like to check availability");
-	String moviename = reader.readLine();
-				 	
-	DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE, MMM dd, yyyy HH:mm:ss a");
-	LocalDate showtime = null;
-	boolean e = false;
-	while(!e) {
-		try {
-			System.out.println("Enter showtime e.g (Saturday, Jul 14, 2018 14:31:06 PM) : ");
-			showtime = LocalDate.parse(reader.readLine(), formatter);									e = true;
-		}catch(DateTimeParseException d) {
-			System.out.println("Invalid date format, Please try again");
-		}	
-	}
+    private void checkSeatAvailability(ArrayList<Cinema> cinemaDB) {
+    	BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+		System.out.println("Enter the name of the Cinema you would like to check seat availability for");
+		String cinemaname = reader.readLine();
+		System.out.println("Enter the Movie for which you would like to check availability");
+		String moviename = reader.readLine();
+					 	
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEEE, MMM dd, yyyy HH:mm:ss a");
+		LocalDate showtime = null;
+		boolean e = false;
+		while(!e) {
+			try {
+				System.out.println("Enter showtime e.g (Saturday, Jul 14, 2018 14:31:06 PM) : ");
+				showtime = LocalDate.parse(reader.readLine(), formatter);									
+				e = true;
+			}catch(DateTimeParseException d) {
+				System.out.println("Invalid date format, Please try again");
+			}	
+		}
+		for (Cinema cinema : cinemaDB) {
+			if (cinema.getName() == cinemaname) {
+				cinema.printLayout(moviename, showtime);
```

Comment:
The args in this call differs from the printLayout(args, args) declareed in the Cinema class. Need to decide on a fix format too